src/main/java/com/powsybl/networkconverterserver/NetworkConverterConstants.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.networkconverterserver;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+
+public final class NetworkConverterConstants {
+
+    private NetworkConverterConstants() {
+    }
+
+    static final String IIDM_CONVERTER_API_VERSION = "v1";
+    static final String CASE_API_VERSION = "v1";
+
+    static final String DONE = "DONE";
```

Comment:
Could you check all these constants are used somewhere and that it is not just a bad copy/paste.